frontend/src/pages/distributedWorkloads/global/projectMetrics/sections/WorkloadResourceMetricsTable.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { Td, Tr } from '@patternfly/react-table';
+import { DistributedWorkloadsContext } from '~/concepts/distributedWorkloads/DistributedWorkloadsContext';
+import { SortableData, Table } from '~/components/table';
+import { WorkloadStatusLabel } from '~/pages/distributedWorkloads/components/WorkloadStatusLabel';
+import { WorkloadResourceUsageBar } from '~/pages/distributedWorkloads/components/WorkloadResourceUsageBar';
+import {
+  WorkloadStatusType,
+  getStatusInfo,
+  getWorkloadRequestedResources,
+} from '~/concepts/distributedWorkloads/utils';
+import { bytesAsPreciseGiB } from '~/utilities/number';
+import { WorkloadKind } from '~/k8sTypes';
+import { ErrorWorkloadState, LoadingWorkloadState, NoWorkloadState } from './SharedStates';
+
+export const WorkloadResourceMetricsTable: React.FC = () => {
+  const { workloads, projectCurrentMetrics } = React.useContext(DistributedWorkloadsContext);
+  const requiredFetches = [workloads, projectCurrentMetrics];
+  const error = requiredFetches.find((f) => !!f.error)?.error;
+  const loaded = requiredFetches.every((f) => f.loaded);
+
+  if (error) {
+    return <ErrorWorkloadState message={error.message} />;
+  }
+
+  if (!loaded) {
+    return <LoadingWorkloadState />;
+  }
+
+  if (!workloads.data.length) {
+    return <NoWorkloadState />;
+  }
+
+  const { getWorkloadCurrentUsage } = projectCurrentMetrics;
+
+  const columns: SortableData<WorkloadKind>[] = [
+    {
+      field: 'name',
+      label: 'Name',
+      sortable: (a, b) =>
+        (a.metadata?.name || 'Unnamed').localeCompare(b.metadata?.name || 'Unnamed'),
```

Comment:
wonder if this should be case insensitive?  if so, maybe
```suggestion
        (a.metadata?.name || 'Unnamed').localeCompare(b.metadata?.name || 'Unnamed', undefined, {sensitivity: 'base'}),
```

Reply:
Ah although actually, these are k8s resources whose names must be all lowercase